x-pack/plugins/osquery/cypress/integration/superuser/packs.spec.ts

```diff
@@ -245,4 +245,21 @@ describe('SuperUser - Packs', () => {
       cy.react('EuiComboBoxInput', { props: { value: '' } }).should('exist');
     });
   });
+  describe.skip('Remove queries from pack', () => {
```

Comment:
skipping because of speed issues like the one above or some other reason?

Reply:
This test is not needed (tbh it doesnt check anything) and was removed in a different PR. However I stopped pulling 'main' due to incoming conflict every week ;p Thanks for bringing my attention to this :)